### ISSUE TITLE
Failed: JMS Connectors should sink disconnect exceptional completion

### ISSUE THREAD

**Issue by ennru** [akka/alpakka#1231]
https://travis-ci.org/akka/alpakka/jobs/432357452

```
[info] JmsConnectorsSpec:
[info] The JMS Connectors
[info] - should sink disconnect exceptional completion *** FAILED *** (90 milliseconds)
[info]   javax.jms.JMSException: Cannot send, channel has already failed: tcp://127.0.0.1:44374 was not an instance of akka.stream.alpakka.jms.ConnectionRetryException, but an instance of javax.jms.JMSException (JmsConnectorsSpec.scala:598)
```

**Comment by ennru** [comment]
Again https://travis-ci.org/akka/alpakka/jobs/443305487

**Comment by 2m** [comment]
One more: https://travis-ci.org/akka/alpakka/jobs/446242926

**Comment by ennru** [comment]
Fixed with #1252
